client/src/app/tabs/MultiSheetTab.js

```diff
@@ -388,6 +388,7 @@ export class MultiSheetTab extends CachedComponent {
         <TabContainer className="content tab">
           <Editor
             ref={ this.editorRef }
+            file={ tab.file }
```

Comment:
I was wondering whether we could simply detect the change from new to not new (`isNew`) so we don't have to pass the entire file to the editor?

Reply:
If we already have the file saved and change the location via `save file as`, we change the location and the potential templates without changes to `isNew`

Reply:
So we're only interested in `path` changes, right?

Reply:
I decided to use the `file` Object and not just the path to check for any save, not just those that change the file path. This allows us to reload the element templates when the bpmn stays in the same place but element templates were added/modified externally.

We can discuss if this is expected behavior